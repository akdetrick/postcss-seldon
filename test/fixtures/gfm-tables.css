/**doc
---
name: Test GFM Tables
---

Col1       | Col2
---------- | -----------
row1 col1  | row1 col2

*/
