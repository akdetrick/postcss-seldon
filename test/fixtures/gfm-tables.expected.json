{
  "meta": {
    "name": "Test GFM Tables"
  },
  "markdown": "\nCol1       | Col2\n---------- | -----------\nrow1 col1  | row1 col2",
  "html": "<table>\n<thead>\n<tr>\n<th>Col1</th>\n<th>Col2</th>\n</tr>\n</thead>\n<tbody><tr>\n<td>row1 col1</td>\n<td>row1 col2</td>\n</tr>\n</tbody></table>\n"
}
